src/AdherentProfile/AdherentProfile.php

```diff
@@ -146,6 +146,17 @@ class AdherentProfile
      */
     private $activityArea;
 
+    /**
+     * @var array
+     *
+     * @Assert\All(
```

Comment:
if I don't do that, It doesn't work. saying that I the value is not in allowed choices. With this, it works. But to be honest, I don't know how it can work on the join page...

Reply:
I see, you can use `multiple=true` attribute on `Choice` constraint

Reply:
indeed, fixed